### PR TITLE
Added heroku ps command

### LINKED ISSUE #75: Add heroku ps command
Add heroku command 
`heroku ps`
This commands shows list of running `dynos`.
It should be added as other commands, following the same structure

### PR DESCRIPTION
**Pull request by alpha-gamma**
Fixes #75 

1. Added heroku ps command with description and examples
2. Added Heroku in Readme file

### CODE REVIEW

File: commands/heroku/heroku-ps.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+`heroku ps {{TYPE}} --app {{APP_NAME}}`
+
+- <b>TYPE</b>: Type of dyno.  This is optional. If it is not given then all the dynos are listed.
+- <b>APP_NAME</b>: Name of the app.
+
+#### Example:
+
+`heroku ps --app TestApp`
+
+=== run: one-off dyno
```
Comment (arshadkazmi42) [inline commands/heroku/heroku-ps.md]: What does this means? As in View mode it looks something like this

screen shot 2019-01-01 at 8 07 14 pm

Comment (arshadkazmi42) [reply commands/heroku/heroku-ps.md]: is this suppose to be in single line?

File: commands/heroku/heroku-ps.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+=== web: bundle exec thin start -p $PORT
+
+web.1: created for 30s
+
+##### with type
+
+`heroku ps web --app TestApp`
+
+=== web: bundle exec thin start -p $PORT
```
Comment (arshadkazmi42) [inline commands/heroku/heroku-ps.md]: Same for this

File: commands/heroku/heroku-ps.md (hunk 1 above)
Comment (alpha-gamma) [reply commands/heroku/heroku-ps.md]: I have moved the run.1 & web.1 to next line

run & web denotes the type of dyno and then the command which created the dyno.
Next line shows the dyno number along with its up time.

Comment (arshadkazmi42) [reply commands/heroku/heroku-ps.md]: So it's basically the response of the command right?
At present we are not planning to put commands output here.
I was thinking of adding a gif of command and output of the command. But haven't started working on it.
If you want you can add the gif, of what happens on executing commands or just the command will be fine.

If you have any inputs on this, I would love to hear it.

File: commands/heroku/heroku-ps.md (hunk 2 above)
Comment (alpha-gamma) [reply commands/heroku/heroku-ps.md]: Updated. All of them were in separate lines in readme file but github rendered it in a single line.

**Comment by arshadkazmi42** [comment]
So it's basically the response of the command right?
At present we are not planning to put commands output here.
I was thinking of adding a gif of command and output of the command. But haven't started working on it.
If you want you can add the gif, of what happens on executing commands or just the command will be fine.

If you have any inputs on this, I would love to hear it.

**Comment by alpha-gamma** [comment]
Okay. For now, I can remove the outputs from it for it to be consistent with others.

We can work separately for adding gif as it will be good to have them for all the commands.

Your thoughts?

**Comment by arshadkazmi42** [comment]
That's what my plan was, but I was not getting time to create gifs. I am planning on adding issues for it.
I am also looking for some collaborators who can help in maintaining this project.
Let me know if you are interested.
You seems to have got hold of the structure we are following.

**Comment by alpha-gamma** [comment]
Looks like a great opportunity. I would like to collaborate.

It will be a good idea to add issues for gif. I can also work on some of the gifs.

**Comment by arshadkazmi42** [comment]
Request sent. You can create issue and review Pull requests

**Comment by alpha-gamma** [comment]
Awesome. 👍

**Pull request merged**